string   id
Position position
---
bool     result
